### PR TITLE
i18n(es): update `install/auto.md` and `install/manual.md`

### PR DESCRIPTION
**Pull request by marsidev**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- New or updated content

#### Description

- What does this PR change? Give us a brief description.
  - Minimal changes to keep it similar to the english version
  - Minimal changes to keep `install/auto.md` and `install/manual.md` similar
    - Like the `auto` version using the word `comenzar` and the `manual` one using `empezar`. Now both uses the same word
  - Change `Typescript` with `TypeScript`
  - Added missing punctuation marks
  - Minimal fixes

Please label this PR with `hacktoberfest-accepted` if get approved. 🎃

### CODE REVIEW

File: src/pages/es/install/manual.md [hunk 1]
```diff
@@ -147,11 +147,11 @@ Si deseas incluir [componentes de frameworks](/es/core-concepts/framework-compon
 
 ## 6. Crea `tsconfig.json`
 
-Typescript es configurado usando `tsconfig.json`. Aún si tú no escribes código en TypeScript, este archivo es importante para que herramientas como Astro y VS Code sepan cómo comprender tu proyecto. Algunas características (como importaciones de paquetes npm) no tienen un soporte completo en el editor sin el archivo `tsconfig.json`.
+TypeScript es configurado usando `tsconfig.json`. Aún si tú no escribes código en TypeScript, este archivo es importante para que herramientas como Astro y VS Code sepan cómo comprender tu proyecto. Algunas características (como importaciones de paquetes npm) no tienen un soporte completo en el editor sin el archivo `tsconfig.json`.
```
Comment (AbrahamX3) [inline src/pages/es/install/manual.md:150]:
```suggestion
TypeScript es configurado usando `tsconfig.json`. Aun si tú no escribes código en TypeScript, este archivo es importante para que herramientas como Astro y VS Code sepan cómo comprender tu proyecto. Algunas características (como importaciones de paquetes npm) no tienen un soporte completo en el editor sin el archivo `tsconfig.json`.
```

**Review by AbrahamX3 — CHANGES_REQUESTED** [review]
¡Buen trabajo! Solo tengo una pequeña sugerencia, me gustaría ver que opinas 🙌

File: src/pages/es/install/manual.md (hunk 1 above)
Comment (marsidev) [reply src/pages/es/install/manual.md:150]: Me parece genial la corrección, no me había fijado del error. Muchas gracias!

**Review by fredoist — APPROVED** [review]
LGTM ✅

Buen trabajo @marsidev

**Comment by marsidev** [comment]
Gracias @fredoist 👌🏼

**Review by AbrahamX3 — APPROVED** [review]

**Pull request merged**